android/guava/src/com/google/common/collect/CompactHashMap.java

```diff
@@ -100,14 +103,39 @@
   private static final Object NOT_FOUND = new Object();
 
   /**
-   * The hashtable. Its values are indexes to the keys, values, and entries arrays.
-   *
-   * <p>Currently, the UNSET value means "null pointer", and any positive value x is the actual
-   * index + 1.
+   * Maximum allowed false positive probability of detecting a hash flooding attack given random
+   * input.
+   */
+  @VisibleForTesting(
```

Comment:
This formatting seems to be slightly off, here.

Reply:
This is from some automated stripping of an internal-only line. Maybe someday we'll update our export process to clean things like this up.